### DISCUSSION TITLE
Cannot run Frigate & script not available in the catalog for fresh install #2709

### DISCUSSION THREAD

**Discussion by vdiogo** [tteck/Proxmox#2710] [Help Needed]
I ran the Frigate installation script 2 months back from here: https://tteck.github.io/Proxmox/

As the Coral USB was not available, and I started reading loads of issues caused by lack of performance without the USB stick, I ordered it and it arrived today (had to be manufactured, shipped, ...).

As I was picking up from where I left the LXC, I cannot get my head around how to run it (the IP is defined as static, but there is no IP defined in the container settings), so I thought on re-starting the process from scratch.

Now I cannot find the installation script anymore on https://tteck.github.io/Proxmox/ however, I do see a file [here](https://github.com/tteck/Proxmox/blob/main/misc/frigate-support.sh) (which I don't know if applicable, a leftover from the previous script, ...).

Anyway, any guidance would be helpful so I can start the "Frigate" process properly.

**Comment by vdiogo** [comment]
In the meanwhile I ran the script [here](https://github.com/tteck/Proxmox/blob/main/misc/frigate-support.sh) with the following command  `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/frigate-support.sh)"` and get to a menu where I'm asked to select the container where to add support.
I see on the list my "old" frigate container, but I don't see my home assistant virtual machine (understand its not a container, but it's where my cameras are configured):
<img width="639" alt="image" src="https://github.com/tteck/Proxmox/assets/24878195/ae9c4448-c7c3-4681-af61-3ad9e5997d91">

**Comment by tteck** [comment]
An installation script for Frigate has never been included in this repository.

**Reply by vdiogo** [reply]
Oh, I could swear it got it form here, but then again, this was done >2 months ago, so I could be completely wrong here.

Anyhow, I do find your scripts super handy, so is the script on your Git something I could leverage from to setup my Frigate?
